src/core/__vendor/launchpad.js

```diff
@@ -8,7 +8,7 @@ export const launchpad = () => {
 			p.GlobalSnowplowNamespace = p.GlobalSnowplowNamespace || [];
 			p.GlobalSnowplowNamespace.push(i);
 			p[i] = function () {
-				(p[i].q = p[i].q || []).push(arguments)
+				(p[i].q = p[i].q || []).push(arguments);
```

Comment:
Good spot! That was me inadvertently at one point having prettier run on the entire `src` directory. I have since removed `src/core/_vendor` from linting. I don't see any impactful changes so I think they will be okay.